Cargo.toml

```diff
@@ -1,12 +1,10 @@
 [workspace]
 members = [
 	"node",
-	"runtime/kreivo",
-	"runtime/virto",
-	"common",
+	"runtime/kreivo"
 ]
 
-exclude = ["xcm-emulator"]
+exclude = ["xcm-emulator",	"common"]
```

Comment:
I don't like excluding modules just because they cause trouble with the tests so I could look into this to see why it gives problem.